src/createNumbersGenerator.js

```diff
@@ -26,7 +26,20 @@
  * @return {number}
  */
 function createNumbersGenerator(numbers, min, max) {
-  // write code here
+  const minItem = min || -Infinity;
+  const maxItem = max || Infinity;
+  let initNumbers;
+  if (numbers) {
+    initNumbers = numbers.filter(item => item >= minItem && item <= maxItem);
+  }
+
+  return function() {
+    let nextItem;
```

Comment:
no need to create this variable, return the result directly

Reply:
> no need to create this variable, return the result directly

refactored